websites/S/SuperiorServers.co/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "Nick",
+    "id": "193909360219389952"
+  },
+  "url": ["forum.superiorservers.co", "superiorservers.co"],
+  "description": {
+    "en": "Presence to show off what you are browsing on SuperiorServers.co to your friends."
+  },
+  "service": "SuperiorServers.co",
+  "version": "1.0",
+  "logo": "https://i.imgur.com/TmuYDD4.png",
+  "thumbnail": "https://i.imgur.com/TmuYDD4.png",
+  "color": "#273341",
```

Comment:
thumbnail should be a promotional banner or a screenshot of the website homepage.